app/src/main/res/values-round/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="hello_world">Hello Round World!</string>

```

Comment:
What it is used for?

Reply:
It must have been for the smart watch... x)